src/main/java/br/com/alura/comex/formatador/Formatacoes.java

```diff
@@ -0,0 +1,26 @@
+package br.com.alura.comex.formatador;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.text.NumberFormat;
+import java.util.Locale;
+
+import br.com.alura.comex.pedido.Pedido;
+
+public class Formatacoes {
+
+	public String formatarValorTotal(BigDecimal montanteDeVendas) {
+		return NumberFormat.getCurrencyInstance(new Locale("pt", "BR"))
+				.format(montanteDeVendas.setScale(2, RoundingMode.HALF_DOWN));
+	}
```

Comment:
No fim das contas, esse código formata um `BigDecimal` como um valor monetário. Poderíamos generalizar dessa maneira:

```suggestion
	public String formatarMoeda(BigDecimal valor) {
		return NumberFormat.getCurrencyInstance(new Locale("pt", "BR"))
				.format(valor.setScale(2, RoundingMode.HALF_DOWN));
	}
```